examples/rifle_sxiv.sh

```diff
@@ -10,39 +10,38 @@
 #
 #   mime ^image, has sxiv, X, flag f = path/to/this/script -- "$@"
 #
-# Implementation notes: this script is quite slow because of POSIX limitations
```

Comment:
I wouldn't remove the comment altogether tbh. Explaining why the script is the way it is would still be good because I could definitely see someone going "Hey, this is way simpler!" in a PR and no one may remember why we didn't use arrays or stored null separated strings in variables.

Reply:
I'm not sure if I would call the script "slow" anymore. How about "For portability reasons the script tries to be POSIX compliant"

As for arrays, I actually tried that. Running it through `Bash` it was actually slower than the POSIX version being run through `Dash`.

Reply:
You don't have to call it slow, rewrite the entire comment if you want.

That's not quite what the comment is saying though. Properly is about doing it correctly not fast. I think it's just explaining how you can't do without the double `listfiles` call. I think we eliminated one of the calls in this PR? Explaining why this was possible and how doing it the other way will cause the problem and require two calls because arrays and null separated strings in vars is not POSIX.

Reply:
Pushed https://github.com/ranger/ranger/pull/2411/commits/0d783e17a461da481a4803e3cf970a15f1500bf5 , is this okay by you?

Reply:
I did not see your message when sending mine. Should I still add notes about the array/null-seperator?

Reply:
Not really what I had in mind. That can be as short as "POSIX compliant."
What I really want is technical details that'll make me think twice before merging a change that'll completely reverse all this work.

Reply:
Oh, one more thing, did you run shellcheck on the script, always a nice sanity check?

Reply:
I see where you're coming from. https://github.com/ranger/ranger/pull/2411/commits/d728452b0ffdb56de3ae642b848477a48f290740 should address it.

Reply:
No, but I just did. And it's all good. :+1: